alphadia/outputtransform.py

```diff
@@ -866,6 +874,14 @@ def _build_run_stat_df(
         if "mobility_fwhm" in channel_df.columns:
             base_dict["fwhm_mobility"] = np.mean(channel_df["mobility_fwhm"])
 
+        with open(optimization_manager_path, "rb") as f:
```

Comment:
are we sure this file is always present or do we need to handle the error case?

Reply:
This is saved by default every time we fit something to the optimization manager, so it should always be present at this stage in the workflow.